docs/apps/cmdArgs.md

```diff
@@ -10,13 +10,13 @@ command line only:
 
 configuration:
   -n [ --name ] arg      name of the broker
-  -t [ --coretype ] arg      type of the broker ("(zmq)", "ipc", "test", "mpi",
```

Comment:
It looks like the command line arguments in the documentation have been reverted to use the (as of v3) old names.